.changeset/better-bushes-make.md

```
@@ -0,0 +1,5 @@
+---
+'@mastra/playground-ui': minor
+---
+
+Added filled and ghost destructive Button variants for dangerous actions.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)